build/scripts/config.js

```diff
@@ -46,6 +46,7 @@ function processBannedWordList(words) {
 }
 exports.bannedWords = {
     normal: processBannedWordList([
+        "Fanum tax", "gyatt", "rizz", "skibidi", //With love, DarthScion
```

Comment:
Super close, but it should be in lowercase... I'll add a comment for the next person.